docs/content/guides/developer/app-examples/blackjack.mdx

```diff
@@ -1,6 +1,101 @@
+
+## Frontend
+
+The [`BlackjackBoard` component](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/app/Blackjack.tsx), a central element of the blackjack game's frontend module, is structured to create an interactive and responsive gaming experience. Written in React, it integrates several features and functions to handle the game's logic and user interactions effectively.
+
+### State management and setup
+
+- **State Hooks:** The module uses React's `useState` to manage various game states, including `playerHand`, `dealerHand`, `playerTotal`, `dealerTotal`, and more. These states are vital for tracking the current game status and updating the UI accordingly.
+
+- **Randomness Generation:** A unique feature of this component is the generation of randomness based on the user's mouse movements (`handleMouseMove`), used to ensure fair game outcomes. The `userCreatedRandomness` state stores this data, which is then converted to a hexadecimal string (`randomnessHex`) for use in the game.
+
+- **Modal for Game Initialization:** A modal is implemented to facilitate the creation of a new game. It captures the randomness and uses `handlePlayGame` from the `useGame` hook to initiate the game with the backend.
+
+- **Card Management:** The component generates a deck of cards and maps them to a `Map` object (`cards`) for easy retrieval and display.
+
+### UI components and styling
+
+- **Card Display:** Cards are rendered using a combination of JSX and CSS, displaying the suit symbols and values, with conditional styling (`getSuitColor`) based on the card suit.
+
+- **Game Status Updates:** The UI provides immediate feedback on game events, such as a bust or game completion, using toasts (`toast`) and loading indicators (`Hearts` from `react-loader-spinner`).
+
+### User interaction and feedback
+
+- **Game Creation and Exploration:** Players can start new games using a dedicated button that triggers the `openNewGameModal` function. Links are also provided for players to view game details on the Sui explorer, enhancing transparency and engagement.
+
+- **Mouse Movement Tracking:** The module includes functionality to track and utilize mouse movements, contributing to the game's randomness generation, which is crucial for fair play and unpredictability.
+
+## Comparison: Blackjack and Coin Flip
+
+### Similarities
+
+- **Blockchain-Based Logic:** Both games are built on Sui, leveraging its capabilities for decentralized applications. The core game logic for each resides in Move modules, ensuring secure and verifiable gameplay.
+
+- **State Management:** In both games, state management is crucial. For Blackjack, this involves managing the player and dealer's hands and scores using React state hooks. In Satoshi Coin Flip, the state is managed through Move structs like `HouseData`, which track the house's balance and other game-related details.
```

Comment:
```suggestion
- **State Management:** In both games, state management is crucial. For blackjack, this involves managing the player and dealer's hands and scores using React state hooks. In Satoshi Coin Flip, the state is managed through Move structs like `HouseData`, which track the house's balance and other game-related details.
```
It's blackjack elsewhere on this page. Not sure which is better but consistency is key.